package.json

```diff
@@ -74,66 +60,74 @@
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react-hooks": "^4.5.0",
-    "fhir": "^4.11.1",
-    "formik": "^2.2.9",
-    "fqm-execution": "^1.3.3",
     "handlebars-loader": "^1.7.2",
+    "html-react-parser": "^1.4.12",
```

Comment:
Shouldn't this be in the dependency section?

Reply:
You are right, updated